wrench/src/reftest.rs

```
@@ -47,6 +65,18 @@ pub fn parse_reftests<F>(filename: &str, mut runner: F)
         };
         let test = dir.join(items.next().unwrap());
         let reference = dir.join(items.next().unwrap());
+
+        if !test.is_file()
```

Perhaps these should return failure / panic, rather than continue?